recipes/bedtools/2.16.2/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: bedtools
+  version: "2.16.2"
+source:
+  fn: BEDTools.v2.16.2.tar.gz
+  url: https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/bedtools/BEDTools.v2.16.2.tar.gz
```

Comment:
If you need an other storage solution, please consider https://github.com/galaxyproject/cargo-port